parser.y

```diff
@@ -791,8 +791,9 @@ import (
 	IfExists			"If Exists"
 	IfNotExists			"If Not Exists"
 	IgnoreOptional			"IGNORE or empty"
-	IndexColName			"Index column name"
+	IndexColName		"Index column name"
 	IndexColNameList		"List of index column name"
+	IndexColNameListOpt	    "List of index column name opt"
```

Comment:
```suggestion
	IndexColNameListOpt		"List of index column name opt"
```

Reply:
![image](https://user-images.githubusercontent.com/19613/62918443-3024a300-bdd2-11e9-8d94-3c55b3ac5708.png)
Please recheck the indentation style. I think I did it right.

Reply:
we usually don't use `space` in yacc file, using `tab`.